ospf6d/ospf6_neighbor.c

```diff
@@ -349,6 +349,18 @@ int negotiation_done(struct thread *thread)
 	return 0;
 }
 
+static int ospf6_neighbor_last_dbdesc_release(struct thread *thread)
+{
+	struct ospf6_neighbor *on;
+
+	on = (struct ospf6_neighbor *)THREAD_ARG(thread);
+	assert(on);
+	on->last_dbdesc_release_timer = NULL;
```

Comment:
this is already NULL.  No need to do it again.

Reply:
fixed